src/configure/configure.ts

```diff
@@ -430,7 +440,6 @@ class Orchestrator {
     }
 
     private async extractAzureResourceFromNode(node: AzureTreeItem | any): Promise<GenericResource> {
-        const resource: GenericResource = null;
         if (!!node.fullId) {
```

Comment:
here you are checking for fullId and in line 267 you have changed this condition

Reply:
The AKS specific work was thee from line 458, where I have updated the condition
```         else if (!!node.cloudResource && node.cloudResource.nodeType === 'cluster') ```

Didn't change this at 434 as not making any difference to us now, and would be better to update if when we are integrating webApp right click scenario for which the code seems to be written.